kernel/kernel-api/src/test/java/io/delta/kernel/internal/replay/TestLogReplay.java

```diff
@@ -0,0 +1,55 @@
+/*
+ * Copyright (2023) The Delta Lake Project Authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.delta.kernel.internal.replay;
+
+import java.util.*;
+
+import org.junit.Test;
+import static org.junit.Assert.*;
+
+import io.delta.kernel.internal.fs.Path;
+import io.delta.kernel.utils.FileStatus;
+
+public class TestLogReplay {
```

Comment:
For tests we made a decision to write them in Scala, because the tests can be very concise in Scala. Could you please convert this into a Scala suite. You can find examples in the `kernel-defaults` module or `kernel-api` module.

Reply:
Just rewrote tests in Scala